samples/docker-domain/Dockerfile

```diff
@@ -39,8 +39,7 @@
 #
 # Pull base image
 # ---------------
-# FROM store/oracle/weblogic:12.2.1.3
-FROM store/oracle/weblogic:12.2.1.3-dev
+FROM container-registry.oracle.com/middleware/weblogic:12.2.1.3-dev
```

Comment:
The Operator references container-registry.oracle.com/middleware/weblogic:12.2.1.3, we should be consistent across our tools